src/rust/engine/process_execution/src/local.rs

```diff
@@ -80,6 +80,7 @@ impl CommandRunner {
       &[],
       StrictGlobMatching::Ignore,
       GlobExpansionConjunction::AllMatch,
+      "".to_string(),
```

Comment:
Ooo, interesting. This might be followup ticket worthy (or TODO). It feels like it would be handy to maybe error for this case...?

But also, it looks like this path could currently be using the simpler constructor for PathGlobs.